tests/functional/external_mode/test_virtualization_storageclass.py

```
@@ -86,52 +83,74 @@ def setup_and_teardown_cnv(self):
+                    f"Step 2: Creating VM '{vm_name}' using SC '{self.virt_sc_name}'."
+                )
+                vm_obj.create_vm_workload(
+                    sc_name=self.virt_sc_name, ssh=False, verify=False
+                )
+                log.info(f"Waiting for VM {vm_name} to reach Running status...")
+                vm_obj.wait_for_vm_status(status=constants.VM_RUNNING, timeout=300)
+            finally:
+                log.info(f"Cleaning up VM resources in {vm_namespace}...")
+                try:
+                    vm_obj.delete()
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)